test-data/unit/check-namedtuple.test

```diff
@@ -35,7 +35,7 @@ X = namedtuple('X', ('x', 'y'))  # type: ignore
 [case testNamedTupleNoUnderscoreFields]
 from collections import namedtuple
 
-X = namedtuple('X', 'x, _y, _z')  # E: namedtuple() field names cannot start with an underscore: _y, _z
+X = namedtuple('X', 'x, _y, _z')  # E: "namedtuple()" field names cannot start with an underscore: _y, _z
```

Comment:
Why did you add the extra quotes? They feel unnecessary.

Reply:
Most of the time mypy highlights the most important code part with `""`, this feels more similar to other cases. For example, here's how it renders in my terminal (with / without `""`):
<img width="752" alt="Снимок экрана 2021-09-18 в 1 20 34" src="https://user-images.githubusercontent.com/4660275/133859617-8bec9b5c-10a7-48e2-91c3-c6d3909c4475.png">

In my opinion, the first one looks and reads better.

Reply:
The double quotes look reasonable to me. Previously we didn't use them probably because the () suffix makes it look like code, it seems okay to add them.